### DISCUSSION TITLE
Sources for the `concourse/dev` image

### DISCUSSION THREAD

**Discussion by virtualstaticvoid** [concourse/concourse#7185] [Help & Support] [answered]
Hello,

I'm trying to figure out where the source `Dockerfile` is for the `concourse/dev` image, as referenced in:

https://github.com/concourse/concourse/blob/master/Dockerfile#L4

Thank you.

**Comment by chenbh** [comment, marked as answer]
That would be in our CI repo: https://github.com/concourse/ci/blob/master/dockerfiles/dev/Dockerfile

**Reply by virtualstaticvoid** [reply]
Awesome, thanks!
